Forwarded: https://github.com/xiph/vorbis/pull/65
From: Matthias Klose <doko@ubuntu.com>
Date: Mon, 11 Oct 2010 10:32:05 +0200
Subject: Fix build failure with DSO link changes.

Make sure to link tests with -lm to fix linker error.

Origin: http://launchpadlibrarian.net/57621522/libvorbis_1.3.1-1_1.3.1-1ubuntu1.diff.gz
Bug-Debian: http://bugs.debian.org/604797
Reviewed-By: Petter Reinholdtsen <pere@debian.org>
Last-Update: 2014-10-14
---
 test/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: libvorbis/test/Makefile.am
===================================================================
--- libvorbis.orig/test/Makefile.am	2018-03-22 08:15:14.098604347 +0100
+++ libvorbis/test/Makefile.am	2018-03-22 08:15:14.094604308 +0100
@@ -10,7 +10,7 @@
 AM_CPPFLAGS = -I$(top_srcdir)/include @OGG_CFLAGS@
 
 test_SOURCES = util.c util.h write_read.c write_read.h test.c
-test_LDADD = ../lib/libvorbisenc.la ../lib/libvorbis.la @OGG_LIBS@ @VORBIS_LIBS@
+test_LDADD = ../lib/libvorbisenc.la ../lib/libvorbis.la @OGG_LIBS@ @VORBIS_LIBS@ -lm
 
 debug:
 	$(MAKE) check CFLAGS="@DEBUG@"
